ansible/roles/zfs_install/tasks/pool.yml

```diff
@@ -25,6 +52,8 @@
         item.state == "present") and
         item.devices[0] not in zpool_devices.stdout and
         item.action|lower == "create"
+  ignore_errors: "{{ ansible_check_mode }}"
```

Comment:
@chriswessels running `ansible-playbook` with --check mode breaks all playbooks that have tasks that depend on a result from a previous task